src/interest-groups/join.js

```diff
@@ -1,3 +1,23 @@
+import {
+	hasInvalidOptionTypes,
+	isMissingRequiredOptions,
+	validateParam,
+} from '../utils/';
```

Comment:
I think to include this in a webpage you need to use `from '../utils/index.js'`

Reply:
Yeah, but I'm still not sure how we want to package this entire lib up right now and so I'm off-putting that til end when its time to optimize and test this in the wild.

Reply:
I won't be able to create an integration test until we can export this code into a browser. We can do something temporary and revisit but whatever temp thing we do needs to work in a browser.

Reply:
Commit: 60938794bd576cb98815e1d4a9d7eaecb70d816e